distribution/pom.xml

```diff
@@ -101,12 +101,13 @@
                         ${basedir}/../serialization/avro/src/main/java;
                         ${basedir}/../backends/jgroups/src/main/java;
                         ${basedir}/../backends/jms/src/main/java;
+                        ${basedir}/../elasticsearch/src/main/java;
                     </sourcepath>
                     <docfilessubdirs>true</docfilessubdirs>
                     <stylesheetfile>${basedir}/src/main/javadoc/stylesheet.css</stylesheetfile>
                     <links>
-                        <link>http://download.oracle.com/javase/7/docs/api/</link>
-                        <link>http://docs.jboss.org/hibernate/orm/5.0/javadocs/</link>
+                        <link>http://download.oracle.com/javase/8/docs/api/</link>
```

Comment:
Yeah, I tried 5.4.4 and it wasn't there so I left it alone. To be honest, I find it less disturbing to have 5.4.0 rather than 5.4.1 if we don't have 5.4.4. But I dont really care :).